easypost/address.py

```diff
@@ -1,12 +1,16 @@
+from typing import Any, Dict
+
 import easypost.easypost_object as util
 from easypost.requestor import Requestor
 from easypost.resource import AllResource, CreateResource
 
 
 class Address(AllResource, CreateResource):
     @classmethod
-    def create(cls, api_key=None, verify=None, verify_strict=None, **params):
-        requestor = Requestor(api_key)
+    def create(
+        cls, api_key: str = None, verify: Dict[str, Any] = None, verify_strict: Dict[str, Any] = None, **params
+    ) -> "Address":
```

Comment:
I'm unfamiliar with the return type of a string of an object, can you point me to docs that define this? Typically I'd expect this to just be `Address` as the object itself and not a string of the word Address.

Reply:
You can type hint a class by name if it's inside of itself or not imported: https://peps.python.org/pep-0484/#forward-references
Otherwise, the type hint would throw an error saying it can't find the referred class.

Reply:
Ah very good, thanks!